packages/edit-post/src/components/sidebar/template/actions.js

```diff
@@ -53,7 +53,7 @@ function PostTemplateActions() {
 	return (
 		<>
 			<div className="edit-post-template__actions">
-				{ !! template && (
+				{ template !== null && (
```

Comment:
Is the change here intentional? What's the reasoning? I'm just trying to think if there are cases where `template` could be `null`?

Reply:
The one I was verifying the operation of got mixed up.

I will revert to `!! template`.